### DISCUSSION TITLE
CReg JDK 11 support for ChemAxon

### DISCUSSION THREAD

**Discussion by jmcneil86043** [mcneilco/acas#990] [Q&A]
We have an ACAS user with ChemAxon JChem for compound reg. They have examined and rejected alternatives to JChem. They need to update to a recent version of Jchem and need to catch up on ACAS releases (they are currently on 1.13.6.6). We received this from ChemAxon support:

> From version 21.19.0 JChem requires Java 11. 
> It does not work with Java 8.
> 
Therefore, this ACAS user requires that we roll acas-roo-server to a newer JDK support Java 11.

What are the blockers?
Thanks, John

**Comment by brianbolt** [comment]
This was implemented here https://github.com/mcneilco/acas-roo-server/pull/349/files. closing disuccion
